data/types/settings/pump/pump.go

```diff
@@ -16,29 +19,50 @@ const (
 	ManufacturersLengthMaximum = 10
 	ModelLengthMaximum         = 100
 	SerialNumberLengthMaximum  = 100
+
+	TimeZoneOffsetMaximum = 7 * 24 * 60 // TODO: Make sure same as all time zone offsets
+	TimeZoneOffsetMinimum = -7 * 24 * 60
+
+	RapidAdult = "rapid_adult"
```

Comment:
```suggestion
	RapidAdult = "rapidAdult"
```
`rapid_adult` is inconsistent with other strings in this PR.